Student/Student/StudentAppDelegate.swift

```diff
@@ -143,22 +141,37 @@ class StudentAppDelegate: UIResponder, UIApplicationDelegate, AppEnvironmentDele
                     .receive(on: RunLoop.main)
                     .sink(receiveCompletion: { _ in }) { brandVars in
                         brandVars.first?.applyBrandTheme()
-                        NativeLoginManager.login(as: session)
                     }
                     .store(in: &self.subscriptions)
             }
+
+            self.setTabBarController()
         }}
     }
 
+    func setTabBarController() {
```

Comment:
Not necessarily in this PR, but would it make sense to implement a superclass like `BaseAppDelegate` to handle the common parts?

Reply:
Yes, it's on the list of future improvements but out of scope of this PR.